selenium_tests/basic_test.py

```diff
@@ -173,3 +179,47 @@ def test_switch_program(self):
         self.wait().until(
             lambda driver: driver.find_element_by_css_selector('.result-info span').text == '{} Results'.format(count)
         )
+
+
+class ReviewFinancialAidTests(SeleniumTestsBase):
+    """Look at the financial aid review page"""
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        Role.objects.create(
+            role='staff',
+            user=cls.user,
+            program=cls.program,
+        )
+
+        cls.student = cls.create_user()
+        tier_program = TierProgram.objects.get(discount_amount=0)
+        FinancialAidFactory.create(
+            user=cls.student,
+            tier_program=tier_program,
+            status=FinancialAidStatus.DOCS_SENT,
+        )
+
+    def setUp(self):
+        super().setUp()
+        self.login_via_admin(self.user)
+
+    def test_approve_docs(self):
+        """Test the financial aid review page"""
+        self.get("/financial_aid/review/{}/{}".format(self.program.id, FinancialAidStatus.DOCS_SENT))
+        self.wait().click(
+            # Mark as received
+            lambda driver: driver.find_element_by_css_selector("tr td:nth-child(7) button")
```

Comment:
unless there's a really compelling reason not to do it, i think we should add a css class or a data- attribute to this button and select for that here rather than going for a child at some numeric position